tools/output-number-of-buffer-segments.rb

```diff
@@ -0,0 +1,49 @@
+#!/bin/ruby
+
+require "open3"
+require "json"
+require "date"
+
+if ARGV.size() == 0
+  puts "Usage: $0 DATABASE_PATH"
+  puts " e.g.: $0 ~/database/db"
+  exit
+end
+
+DATABASE_PATH = ARGV[0]
+ERROR_LOG_NAME = "output_n_buffer_segments.error"
+
+def output_error(status, stdout, stderr)
+  File.open(ERROR_LOG_NAME, 'a') do |file|
```

Comment:
Why don't you just output error message to the standard error?

Reply:
I predict that this tool is executed periodically with crontab and so on.
Therefore, I think that It is easier to use its output to file rather than output to stdout.

Reply:
If we use crontab, it can e-mail automatically when command outputs something.
In this case, users don't need to check error log file on remote machine.

Reply:
Right.
I've changed that error log output to stdout.